### DISCUSSION TITLE
Raman gain coefficient query for fibers other than SSMF

### DISCUSSION THREAD

**Discussion by mukherjeearup** [Telecominfraproject/oopt-gnpy#503] [Q&A] [answered]
Hi GnPy Team,
So far I can see data available only for SSMF Raman coefficient profile normalized with respect to the effective area overlap (g0 * A_eff(f_probe, f_pump)) at ref wavelength 1454 nm.
Don't we support Raman solver for any other type of fiber like G655 in GnPy?
If yes, can you pls point out to the g0 coefficients for G655 fiber?

Regards,
Arup

**Comment by AndreaDAmico** [comment, marked as answer]
Dear @mukherjeearup,
In GNPy, the exact same Raman coefficient shape when the user does not provide a measured g0. The Raman coefficient shape is rescaled depending on the effective area of the fiber under consideration. In general, the material concentrations in different fiber types vary the Raman coefficient shape and a simple rescaling is not enough to model this variation. It would be very useful if the GNPy community could collect and share measured Raman coefficients for a larger set of fiber types.

Please note that the current GNPy version contains a bug in the effective area scaling of g0. A fix is already under review.

Regards,
Andrea

**Reply by mukherjeearup** [reply]
Hi @AndreaDAmico ,
Thanks for your reply.
May I know the exact bug/issue description in the effective area scaling in current GnPy? Shall it be fixed in the next release?

Regards,
Arup

**Reply by AndreaDAmico** [reply]
The issue arises when the default Raman coefficient is used and the fiber effective area parameter is different from 83e-12 m^2.
In general, to obtain the normalized Raman coefficient g0 must be multiplied by the effective area evaluated at the pump frequency that has been used for the Raman coefficient measurement (206 THz in the case of the default value). In GNPy, the default g0 refers to a measurement with an SSMF and a 206 THz pump frequency; it must always be normalized with respect to the effective area evaluated at the pump frequency in the SSMF case, i.e. the scaled effective area evaluated at 206 THz w.r.t 83e-12 m^2 at 193.4 THz. Instead, the current GNPy version normalizes g0 with respect to the effective area evaluated at the pump frequency scaling the provided new effective area.

We will public the fix shortly, for sure it will be included in the next release.

**Reply by mukherjeearup** [reply]
Understood @AndreaDAmico . Thanks for your quick reply!
